src/shared/components/Buttons.js

```diff
@@ -0,0 +1,120 @@
+import classnames from 'classnames';
```

Comment:
The fact that this module name begins with a capital letter suggests to me that it exports a single component called `Buttons`. I suggest that for modules which export multiple components, we fall back to our normal module naming scheme (hyphenated).

Reply:
This point is good. I'll rename the module.